server/src/main/java/org/elasticsearch/index/mapper/MappingLookup.java

```diff
@@ -223,6 +225,14 @@ FieldTypeLookup fieldTypesLookup() {
      * Returns the total number of fields defined in the mappings, including field mappers, object mappers as well as runtime fields.
      */
     public long getTotalFieldsCount() {
+        return totalFieldsCount;
+    }
+
+    /**
+     * Returns the total number of mappers defined in the mappings, including field mappers and their sub-fields
+     * (which are not explicitly defined in the mappings), multi-fields, object mappers, runtime fields and metadata field mappers.
+     */
+    public long getTotalMapperCount() {
```

Comment:
I think I can no longer tell the difference between getTotalFieldsCount and getTotalMapperCount . It's very subtle, or is there any difference at all?

Reply:
See https://github.com/elastic/elasticsearch/pull/102885#issuecomment-1929207955